packages/plugin-ext/src/common/plugin-api-rpc-model.ts

```diff
@@ -660,6 +660,8 @@ export interface Comment {
     readonly contextValue?: string;
     readonly label?: string;
     readonly mode?: CommentMode;
+    /** Timestamp serialized as ISO date string via Date.prototype.toISOString */
+    readonly timeStamp?: string;
```

Comment:
minor: `timeStamp` > `timestamp`

```suggestion
    readonly timestamp?: string;
```

Reply:
This is solved with commit 376dbca. time**s**tamp is now used.